aggregate/PUMS/pums_0812_1519_demographics.py

```diff
@@ -0,0 +1,50 @@
+"""This script takes the xlsx files Erica from DCP Population sent us imports them, 
```

Comment:
There are columns in the final dataframes that are missing. In the source data, erica provides columns `pop3_12/pop3_19` and `pop1_12/pop1_19` which both refer to total population (the denominator for the foreign born poulation and then the denominator for age). Neither of these make it into the final data that is accessed via the main accessor `acs_pums_demographics`. I believe these are duplicates but weren't caught because they are unique column names. First step is to drop one of them, update them to follow convention (i.e. `pop1_12e` : `pop_0812_count`, etc), and then include them in the final output. the reason they aren't making it into the final dataframe is the reorder function, I think you need to include pop in the categories for it to be captured in the final output

Reply:
got it. I only include the columns that were included in the earlier demographics work probably was a mistake. So to reiterate what you said so I have it correct, I will only need only include one set i.e. `pop1` and for both year and convert to convert to our naming convention to get the correct output

Reply:
Yeah it was a mistake on my end as well - the issue is with the reordering